hash/oric1_cass.xml

```diff
@@ -1755,6 +1744,20 @@ Tape 2 Side B - VOYAGES: Which is loaded when you go from one town to another
 		</part>
 	</software>
 
+	<software name="galaxy5">
+		<description>Galaxy 5</description>
+		<year>1983</year>
+		<publisher>Durell Software</publisher>
+		<notes><![CDATA[
+Only has the "Galaxy" game. Missing "Lunar", "Astro-War", "Space-Chase", "Asteroids".
+]]></notes>
+		<part name="cass" interface="oric1_cass">
+			<dataarea name="cass" size="8134">
+				<rom name="galaxy.tap" size="8134" crc="a96b6f8e" sha1="bd4eafb53fab98d739868a341f83d45752d4c3b7"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Is this missing additional tapes, or is it a bad dump?  Either way, it should probably be marked (either with placeholders for missing tapes or marked as a bad dump).

Reply:
Marked as bad dump.